src/foundry/client/apps/sidebar/package-configuration.d.ts

```diff
@@ -0,0 +1,97 @@
+export {};
+
+/**
+ * An application for configuring data across all installed and active packages.
+ */
+declare global {
+  abstract class PackageConfiguration<ConcreteCategory extends PackageConfiguration.Category> extends FormApplication<
+    PackageConfiguration.Options,
+    PackageConfiguration.Data<ConcreteCategory>
+  > {
+    static get categoryOrder(): string[];
+
+    /**
+     * @defaultValue
+     * ```typescript
+     * foundry.utils.mergeObject(super.defaultOptions, {
+     *   classes: ["package-configuration"],
+     *   template: "templates/sidebar/apps/package-configuration.html",
+     *   categoryTemplate: undefined,
+     *   width: 780,
+     *   height: 680,
+     *   resizable: true,
+     *   scrollY: [".filters", ".categories"],
+     *   filters: [{inputSelector: 'input[name="filter"]', contentSelector: ".categories"}],
+     *   submitButton: false
+     * });
+     * ```
+     */
+    static override get defaultOptions(): PackageConfiguration.Options;
+
+    override getData():
+      | PackageConfiguration.Data<ConcreteCategory>
+      | Promise<PackageConfiguration.Data<ConcreteCategory>>;
+
+    abstract _prepareCategoryData(): PackageConfiguration.Categories<ConcreteCategory>;
+
+    /**
+     * Classify what Category an Action belongs to
+     * @param namespace - The entry to classify
+     * @returns The category the entry belongs to
+     */
+    protected _categorizeEntry(namespace: string): PackageConfiguration.Category;
+
+    /** Reusable logic for how categories are sorted in relation to each other. */
+    protected _sortCategories(a: PackageConfiguration.Category, b: PackageConfiguration.Category): number;
+
+    protected override _render(
+      force?: boolean | undefined,
+      options?: Application.RenderOptions<PackageConfiguration.Options> | undefined
+    ): Promise<void>;
+
+    override activateListeners(html: JQuery<HTMLElement>): void;
+
+    /**
+     * Handle left-click events to filter to a certain category
+     * @internal
+     */
+    protected _onClickCategoryFilter(event: MouseEvent): void;
+
+    protected override _onSearchFilter(event: KeyboardEvent, query: string, rgx: RegExp, html: HTMLElement): void;
+
+    /**
+     * Handle left-click events to show / hide a certain category
+     * @internal
+     */
+    protected _onClickCategoryCollapse(event: MouseEvent): void;
+
+    /**
+     * Handle button click to reset default settings
+     * @param event - The initial button click event
+     */
+    protected _onResetDefaults(event: Event): void;
```

Comment:
Technically, `event` also is always a `JQuery.ClickEvent`, and the jsdoc also suggests that it indeed only should be click events. So I'd use that, even if the documentation inf foundry is just `Event`. Might be worth reporting to them.